Example/Core/Tests/FIRAppTest.m

```diff
@@ -676,6 +676,7 @@ - (void)testAnalyticsNotSetByGlobalDataCollectionSwitch {
   OCMReject([configurationMock setAnalyticsCollectionEnabled:YES persistSetting:NO]);
   [app setDataCollectionDefaultEnabled:YES];
 
+  OCMReject([configurationMock setAnalyticsCollectionEnabled:YES persistSetting:YES]);
```

Comment:
I believe we'd want to reject `setAnalyticsCollectionEnabled:NO persistSetting:YES` here since we're setting a `NO`.